pkg/apis/config/feature_flags.go

```diff
@@ -76,6 +82,8 @@ const (
 	DefaultResourceVerificationMode = SkipResourceVerificationMode
 	// DefaultEnableProvenanceInStatus is the default value for "enable-provenance-status".
 	DefaultEnableProvenanceInStatus = false
+	// DefaultCustomTaskVersion is the default value for "custom-task-version"
+	DefaultCustomTaskVersion = CustomTaskVersionBeta
```

Comment:
Based on [TEP-0114 New Feature Flag `custom-task-version`](https://github.com/tektoncd/community/blob/main/teps/0114-custom-tasks-beta.md#new-feature-flag-custom-task-version)
> We set the default feature flag value to `v1alpha1` in the first LTS release, and switch to `v1beta1` in the following three LTS releases.

We may want to set the default value to `v1alpha1` in the upcoming release `v0.43.0`, switch the default value to `v1beta1` in the release `v0.44.0`, and delete the feature flag in the release `v0.47.0`

Reply:
Ok.

Reply:
I'll get to this later today, 'cos I have to rewrite a _lot_ of tests. =)

Reply:
NP~ Thank you! ❤️